lemminx-maven/src/main/java/org/eclipse/lemminx/maven/MavenHoverParticipant.java

```diff
@@ -154,10 +157,23 @@ private Hover collectArtifactDescription(IHoverRequest request, boolean isPlugin
 				.findFirst()
 				.map(localRepoSearcher::findLocalFile)
 				.map(file -> builder.buildRawModel(file, ModelBuildingRequest.VALIDATION_LEVEL_MINIMAL, false).get())
-				.map(model -> model.getName() + "\n\n" + model.getDescription())
+				.map(model -> {
+					UnaryOperator<String> toBold = message -> supportsMarkdown ? MarkdownUtils.toBold(message) : message;
```

Comment:
This can probably be replaced by `UnaryOperator<String> toBold = supportsMarkdown ? MarkdownUtils::toBold : UnaryOperator.identity()`

Reply:
Since the identity function returns an Object, should I just cast it to a String?

Reply:
identity() returns a `UnaryOperator<T>` so the type should be properly inferred by the compiler to `UnaryOperator<String>`.

Reply:
When I change it, I get a ```Type mismatch: cannot convert from UnaryOperator<Object> to String```. Should I just ignore it?

Reply:
This works for me locally. Are you using the exact same syntax I mentioned earlier?

Reply:
So sorry, that's my bad, all working now.